### PR TITLE
Update neutron api doc with horizon notes.

### PR DESCRIPTION
**Pull request by Lukasa**

### CODE REVIEW

File: docs/source/calico-neutron-api.rst [hunk 1]
```diff
@@ -144,3 +153,69 @@ to create one will fail.
...
+Tab: Network -> Networks
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For networks and subnets, see the sections :ref:`neutron-api-networks` and
+:ref:`neutron-api-subnets`.
+
+Tab: Network -> Routers
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Tenants should not be able to create routers, as they serve no purpose in a
```
Comment (ed-harrison) [inline docs/source/calico-neutron-api.rst]: What do you mean by "should not be able to"?  Is this something that Horizon can be set to police (through quotas) or did you really mean "should not as the attempt will fail and it is pointless"?

File: docs/source/calico-neutron-api.rst [hunk 2]
```diff
@@ -144,3 +153,69 @@ to create one will fail.
...
+
+In the course of general operation administrators are not expected to make
+changes to their networking configuration. However, for initial network setup,
+this panel may be used to make changes. See :doc:`opens-external-conn` for
+details on how to achieve this setup.
+
+Tab: System Panel -> Routers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Administrators should not create routers, as they serve no purpose in a Calico
```
Comment (ed-harrison) [inline docs/source/calico-neutron-api.rst]: Maybe use the text here in the section on Network -> Routers, above?

**Comment by Lukasa** [comment]
Alright @neiljerram, I've made those changes. =)

**Comment by neiljerram** [comment]
Thanks.

**Pull request merged**